### PR TITLE
TT-9: added business logic for creating time records

### PR DESCRIPTION
**Pull request by x13enos**
https://trello.com/c/vSjGWzPQ/12-tt-9-add-timerecord

### CODE REVIEW

File: test/fixtures/time_records.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (volonterx) [inline test/fixtures/time_records.yml]: I couldn't find where are this fixtures are used. 
If you describing two object with the same attributes, you should use some kind of template instead.

File: app/graphql/mutations/time_records/create.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        time_record = create_record(data)
+        return { time_record: time_record }
+      rescue ActiveRecord::RecordInvalid => e
+        GraphQL::ExecutionError.new(e)
+      end
+
+      private
+
+      def create_record(data)
+        record_params = data.to_h.merge(time_start: Time.now)
```
Comment (volonterx) [inline app/graphql/mutations/time_records/create.rb]: In tests I found an example of request. In this request spentTime is defined. 
So record will be created for Time.now and in will end in the future. 
But what if I send another record with spentTime in several minutes, when several minutes < that spentTime of first record? We will have to tasks that intersects. 

Also the force rewriting of time_start here it's a bad thing. Maybe Time.now should be default value. Or ability to send spend_time via creating should be forbidden.

Maybe it's all species of client, but than it should declare all information about record, not backend. And backend should validate intersects of records, I think.

Comment (x13enos) [reply app/graphql/mutations/time_records/create.rb]: I think we clarified this issue. We also can complex the logic of tracking time later.

File: test/fixtures/time_records.yml (hunk 1 above)
Comment (x13enos) [reply test/fixtures/time_records.yml]: pretty sure it was created by some kind of generators, Rspec or bin/rails maybe

**Pull request merged**